vmware-base/src/com/cloud/hypervisor/vmware/util/VmwareContext.java

```diff
@@ -375,6 +377,18 @@ public void uploadFile(String urlString, File localFile) throws Exception {
         }
     }
 
+    private Charset getCharSetFromConnection(HttpURLConnection conn) {
```

Comment:
We should move this to one of the util classes. I am sure it can be used in more places then just here.